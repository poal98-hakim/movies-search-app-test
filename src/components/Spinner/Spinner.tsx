import React from "react";
import "./Spinner.scss";

const Spinner : React.VFC = () => <div className="loader" data-testid="loader"></div>;

export default Spinner;
