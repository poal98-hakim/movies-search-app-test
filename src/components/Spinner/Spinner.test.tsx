import Spinner from "./Spinner";
import { render } from '@testing-library/react';


describe('Spinner', () => {

  beforeEach(() => { //before each test render the component and reset the value of props
    render(<Spinner />);
  });
  
  it('renders correctly', () => {
    const { baseElement } = render(<Spinner />);
    expect(baseElement).toMatchSnapshot()
  })
  
})
