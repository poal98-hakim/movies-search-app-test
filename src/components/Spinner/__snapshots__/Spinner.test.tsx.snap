// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner renders correctly 1`] = `
<body>
  <div>
    <div
      class="loader"
      data-testid="loader"
    />
  </div>
  <div>
    <div
      class="loader"
      data-testid="loader"
    />
  </div>
</body>
`;
